projects/js-packages/publicize-components/src/components/settings-button/index.js

```diff
@@ -14,6 +19,62 @@ import styles from './styles.module.scss';
  * @returns {object} The link/button component.
  */
 export default function PublicizeSettingsButton() {
+	const useAdminUiV1 = useSelect( select => select( SOCIAL_STORE_ID ).useAdminUiV1() );
+
+	const [ currentService, setCurrentService ] = useState( null );
+	const [ isModalOpen, toggleModal ] = useReducer( state => ! state, false );
+
+	return useAdminUiV1 ? (
+		<ThemeProvider targetDom={ document.body }>
+			<button
+				className={ styles[ 'settings-link' ] }
+				onClick={ toggleModal }
+				title={ __( 'Connect an account', 'jetpack' ) }
```

Comment:
The `title` is not good/enough for a11y. We should always add `aria-label` when a button doesn't have text.

> Relying on the `title` attribute is currently discouraged as many user agents do not expose the attribute in an accessible manner as required by this specification (e.g., requiring a pointing device such as a mouse to cause a tooltip to appear, which excludes keyboard-only users and touch-only users, such as anyone with a modern phone or tablet).

[W3C spec](https://www.w3.org/TR/html/dom.html#the-title-attribute)